ferrilata-devilline/Controllers/PitchController.cs

```diff
@@ -34,5 +34,24 @@ public IActionResult Return_Pitches()
 
             return Unauthorized(new Error("Unauthorized"));
         }
+
+        [HttpPut("pitch")]
+        public IActionResult PutPitch([FromBody] Pitch pitchToUpdate)
+        {
+            var request = Request;
```

Comment:
41: var request = Request;
Not necessary to create var request, just use Request in ifs below